src/main/java/com/sublinks/sublinksapi/language/services/LanguageService.java

```diff
@@ -34,4 +34,11 @@ public List<Language> languageIdsToEntity(final Collection<String> languageIds)
     }
     return languages;
   }
+
+  public List<Language> instanceLanguages(final Instance instance) {
+
+    final List<Language> languages = new ArrayList<>();
+    languages.addAll(instance.getLanguages());
+    return languages;
+  }
```

Comment:
This method makes zero sense to me. It's taking a list and making a list from it, right?